compiler/semstmts.nim

```diff
@@ -1890,4 +1890,6 @@ proc semStmtList(c: PContext, n: PNode, flags: TExprFlags): PNode =
 
 proc semStmt(c: PContext, n: PNode): PNode =
   # now: simply an alias:
+  pushInfoContext(c.config, n.info)
```

Comment:
Dubious, and likely to produce tons of false positives. The error message is **Instantiation from here** and ``semStmt`` doesn't perform any instantiations. Instantiation here refers to template/macro or generic expansions.

Reply:
how about renaming "template/generic instantiation from her" to something more general, eg "obtained from her" ?

> Dubious, and likely to produce tons of false positives

do you have such an example where it produces a ton of false positives?

Reply:
Well the tests are not even green with this PR, for example tests/parallel/tgc_unsafe2.nim fails.

Reply:
right, they would be green if I make these context subject to a `hint[extendedContext]` (which would be off by default); If you're ok with that I'll go ahead and make that change.